### ISSUE TITLE
PHP 7.4 compatability

### ISSUE THREAD

**Issue by tabennett** [yooper/php-text-analysis#66]
Is there any way that this package could be updated to require wamania/php-stemmer ~2 (instead of ~1) for php 7.4 compatibility? I can PR this if you'd like.

**Comment by yooper** [comment]
Hello, yes, I would definitely appreciate the contribution. Please make sure the unit tests pass before submitting a PR. 

Thanks,

**Comment by tabennett** [comment]
Sorry I'm late on this; I mean to get started on this earlier but the past few months since I opened this issue have been pretty crazy.  I've forked this Repo now and would like to help get this compatible with php 7.4 and 8.x if you still want the help. I see you're already working on some 7.4 stuff and I don't wanna step on your toes before I PR this.

I've installed composer dependencies and ran the test suite but there were failure from missing directories. Looked around a bit and it seems like I need to install some pta data packages using the textconsole. So I've got those installing now. Is there anything else I'll need to run these? I already have java installed too (saw that on the wiki).
